test/WhiteBoxTests.cpp

```diff
@@ -945,7 +951,7 @@ void WhiteBoxTests::testTime()
     t = std::chrono::system_clock::time_point();
     LOK_ASSERT_EQUAL(std::string("Thu, 01 Jan 1970 00:00:00"), Util::getHttpTime(t));
 
-    t = std::chrono::system_clock::time_point(std::chrono::nanoseconds(1569592993495336798));
+    t = std::chrono::system_clock::time_point(std::chrono::duration_cast<std::chrono::system_clock::duration>(std::chrono::nanoseconds(1569592993495336798)));
```

Comment:
I imagine there is a shortcut to this, but I haven't figured it out.

Btw, worth using clang-format here (and everywhere, really).